app/components/avo/views/resource_index_component.rb

```diff
@@ -164,7 +164,8 @@ def scopes_list
       resource: resource,
       turbo_frame: turbo_frame,
       parent_record: parent_record,
-      query: query
+      query: query,
+      remove_scope_all: resource.entity_loader(:scope).remove_scope_all
```

Comment:
Can't we pass the loader and have the component ask the loader if it should show the default scope or not?

I feel that adding arbitrary options to it every time we change the API is not the way to go.

Reply:
We can and it's better for scalability